app/wizards/schools/register_ect_wizard/start_date_step.rb

```diff
@@ -21,6 +22,32 @@ def previous_step
 
     private
 
+      def start_date_not_before_previous_school
+        return if skip_start_date_validation?
```

Comment:
to avoid showing two error messages, can we add `return if errors.any?`

Reply:
It's already in the `skip_start_date_validation?` method (line 47)

```
 def skip_start_date_validation?
        start_date.blank? || errors[:start_date].any?
      end
```